subsys/compress/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library(nrf_compress)
+zephyr_include_directories(include)
+zephyr_library_sources(src/implementation.c)
+zephyr_linker_sources(SECTIONS sections.ld)
+zephyr_iterable_section(NAME nrf_compress_implementation KVMA RAM_REGION GROUP RODATA_REGION SUBALIGN CONFIG_LINKER_ITERABLE_SUBALIGN)
```

Comment:
shouldn't this be `zephyr_linker_section` ? The `zephyr_iterable_section` does not seem to take KVMA parameter.

Reply:
?? https://github.com/zephyrproject-rtos/zephyr/blob/main/cmake/linker_script/common/common-rom.cmake#L14

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/c008cbab1a05316139de191b0553ab6ccc0073ad/cmake/modules/extensions.cmake#L4923 ?